src/main/java/org/broadinstitute/consent/http/db/mapper/DatasetReducer.java

```diff
@@ -29,6 +29,9 @@ public void accumulate(Map<Integer, Dataset> map, RowView rowView) {
       dataset.setDataUse(
           DataUse.parseDataUse(rowView.getColumn("data_use", String.class)).orElse(null));
     }
+    if (hasColumn(rowView, "translated_data_use", String.class)) {
+      dataset.setTranslatedDataUse(rowView.getColumn("translated_data_use", String.class));
+    }
```

Comment:
On the whole I'm not a huge fan of `Optional<T>` but in this case I think it would be a good improvement. The code currently calls `getColumn()` twice and has to repeat itself when specifying the name and type of the column. Adding a `default` method to `RowMapperHelper` that returns an `Optional` would avoid both of these issues and then this code would look like:
```
    getColumn(rowView, "translated_data_use", String.class).ifPresent(dataset::setTranslatedDataUse);
```

This method should work:
```
  default <T> Optional<T> getColumn(RowView rowView, String columnName, Class<T> clazz) {
    try {
      return Optional.of(rowView.getColumn(columnName, clazz));
    } catch (Exception e) {
      log.debug("RowView does not contain column " + columnName);
      return Optional.empty();
    }
  }
```

Reply:
Yes, I like that - will take another stab at it.